### ISSUE TITLE
paypal screen not displayed

### ISSUE THREAD

**Issue by mobil-app-explorer** [jibon57/nativescript-braintree#39]
The plug in is very helpful, thanks very much! I got it work for credit card payment on android. But if I click on paypal option instead of credit card option, I see a busy cursor keep spinning, when I click on it, the process got cancelled and returned to my main screen, so I never get to paypal screen, I did set intent-filter in AndroidManifest.xml, what might be the solution for this? Thanks in advance!

**Comment by mobil-app-explorer** [comment]
I downloaded plug in project, and executed demo.android script to check demo, still same issue, paypal screen not showing up on android.

**Comment by felixkrautschuk** [comment]
Unfortunately, I am not able to reproduce your issue. The paypal screen is showing up correctly for me on our own app and the demo app of this repo. I tested on Android 5 Emulator and on my Pixel 3a with Android 11.

**Comment by mobil-app-explorer** [comment]
Thanks for taking a look at this! The issue happens on my real device Pixel 3a and Android 11 phone, here is the sequence of operation, let me know if I did anything incorrectly since I am beginner for mobile development: 
1. git clone the project, 
2. in windows command, go to "src" directory, and issue "npm install", then "npm run tsc", then "cd ../demo", then "tns run android"
3. then I see "Pay" button in my android screen, click on that, then I see two options "PayPal" and "Credit or Debit Card", click on "PayPal", then I got the spinning cursor....

**Comment by mobil-app-explorer** [comment]
One update, after I bumped drop-in version to 5.2.0 "com.braintreepayments.api:drop-in:5.2.0", it is working now.

**Comment by felixkrautschuk** [comment]
Could you check one more time if the issue occurs with the demo app of this repo on your side?
I noticed the issue on my side maybe 3 or 4 days ago, but without any changes it is working as expected now. So I'm sure it is not related to the drop-in dependency version, maybe there was a problem on Braintree itself or Paypal...

**Comment by mobil-app-explorer** [comment]
Thanks for looking into this! In order to rule out that I accidentally modified anything, I did a fresh start from git clone again, and I was able to reproduce the issue, after I changed the plugin version to 5.2.0, it worked again, then I changed back to 4.5.0, the issue reoccurred again. It may not be problem of the drop-in dependency version, but seems in my case, newer version at least providing a workaround. Let me know if any other info/testing I can provide.
